test/unit/rules/lint-table-groups-test.js

```diff
@@ -316,5 +356,15 @@ generateRuleTests({
         column: 0,
       },
     },
+    {
+      template: '<table>some text</table>',
```

Comment:
Previously this would have been accepted, since the check in `hasChildTags` didn't have the whitespace check. There just weren't any tests for it.